frontend/src/features/admin-form/responses/IndividualResponsePage/IndividualResponsePage.tsx

```diff
@@ -157,6 +158,12 @@ export const IndividualResponsePage = (): JSX.Element => {
     ? getStatusFromWorkflowStatus(workflowStatus)
     : ''
 
+  const lastSubmittedAt = data?.mrf?.lastSubmittedAt
+    ? moment(data.mrf.lastSubmittedAt)
```

Comment:
this is necessary as individual currently includes `ddd, ...` in the front while the table and CSV mode do not. this is the cleanest way which delegates to the FE for formatting.